sycl/include/sycl/ext/oneapi/experimental/bfloat16.hpp

```diff
@@ -50,8 +63,10 @@ class bfloat16 {
     return __spirv_ConvertBF16ToFINTEL(a);
 #endif
 #else
-    throw exception{errc::feature_not_supported,
-                    "Bfloat16 conversion is not supported on host device"};
+    // Shift temporary variable to silence the warning
+    uint32_t bits = a;
```

Comment:
Can we use fast CPU builtins here rather than the slow software conversion?

Reply:
No, but I am referring to the builtins used in __spirv_ConvertBF16ToFINTEL's lowering on the CPU device?

Reply:
could the host be non-x86? if so, i think we can't use built-in for specific platform

Reply:
We shouldn't use SPIR-V instructions in host device implementation as:
1. non-x86 platform can be a host device;
2. not all x86 platforms support the instruction/builtin;
3. host compiler is not necessarily Intel-built clang compiler.

Reply:
I don't know of any such builtins. @dkhaldi - Do you know of any public and documented x86-builtins we could use here? As @YetAnotherCompilerEngineer said, I don't believe we do any lowering of `__spirv_ConvertBF16ToFINTEL` on host.

Reply:
We can't call any specific x86 buildins here since we don't have them.